named_export.js

```diff
@@ -0,0 +1,10 @@
+
+export const sqrt = Math.sqrt;
+export function square(x) {
+  return x * x;
+}
+export function diag(x, y) {
+  return sqrt(square(x) + square(y));
+}
+console.log(square(5)); 

```

Comment:
again what the purpose to use your functions here -- you should import them and use in the main file